helmfile.d/snippets/defaults.yaml

```diff
@@ -23,6 +23,8 @@ environments:
               limits:
                 cpu: 200m
                 memory: 384Mi
+          cloudnative-pg:
```

Comment:
Since the `cloudnative-pg`  is always enabled this default is not necessary.

Reply:
Fixed in: https://github.com/redkubes/otomi-core/pull/1137/commits/19d1d8dde426694bffee41dd3b0a56cc7fabb904